page/using-jquery-core/attributes.md

```diff
@@ -1,27 +0,0 @@
-<script>{
```

Comment:
I think that merging the two pages is great but I'm a bit worried about broken links. I think this page should redirect to the "Manipulating Attributes" section of the other page. Any thoughts?